frontend/src/components/SignInButton.tsx

```diff
@@ -24,8 +24,11 @@ export const SignInButton = () => {
 
   return (
     <Button
-      variant="outlined"
+      variant="contained"
       className="ml-auto"
+      size="large"
+      sx={{ mx: 'auto' }}
+      color="secondary"
       onClick={() => handleLogin(instance)}
       disabled={isLoading}
```

Comment:
On mobile it seems that this comes outside of the blue background. You can see what this looks like on mobile by opening developer tools (hit F12). And then hit this button on the left
<img width="1510" alt="Screen Shot 2022-10-10 at 5 41 04 pm" src="https://user-images.githubusercontent.com/24561449/194810433-f7ad7ff4-0dc3-49c3-8094-a239b5d7c989.png">

Feel free to change any of the code I've written in components

Reply:
Ah damn. Will look to fix that.